recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
I'm not really sure how your installation work. But in the end your binaries needs to end up in $PREFIX/bin. You are not allowed to change $PREFIX :(